### PR TITLE
[ADF-4066] Should create a task unassigned when assignee field is empty …

### PR DESCRIPTION
**Pull request by gmandakini**
…in Start Task form automated

**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4129

### CODE REVIEW

File: e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts [hunk 1]
```diff
@@ -23,11 +23,13 @@ import { NavigationBarPage } from '../pages/adf/navigationBarPage';
 import { TasksCloudDemoPage } from '../pages/adf/demo-shell/process-services/tasksCloudDemoPage';
 import { StartTasksCloudComponent } from '../pages/adf/process-cloud/startTasksCloudComponent';
 import { Util } from '../util/util';
+import {TaskDetailsPage} from "../pages/adf/demo-shell/process-services/taskDetailsPage";
 
 describe('Start Task', () => {
 
     const settingsPage = new SettingsPage();
     const loginSSOPage = new LoginSSOPage();
+    let taskDetailsPage = new TaskDetailsPage();
```
Comment (eromano) [inline e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts]: can be a const

Comment (eromano) [reply e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts]: there are also some lint error

Comment (gmandakini) [reply e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts]: fixed, thanks

**Comment by gmandakini** [comment]
I still need to run the tests properly, unable to do it atm, as the required apps creation is still in progress on aps2staging.

File: e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { by, element } from 'protractor';
+import { Util } from '../../../../util/util';
+
+export class TaskDetailsPage {
+
+    taskDetailsHeader = element(by.css('ng-component h4'));
```
Comment (eromano) [inline e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts]: Is it possible find a better hook? like an id?

File: e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts [hunk 3]
```diff
@@ -129,4 +131,24 @@ describe('Start Task', () => {
         tasksCloudDemoPage.taskListCloudComponent().getDataTable().checkContentIsDisplayed(standaloneTaskName2);
     });
 
+    it('[C297675] Should create a task unassigned when assignee field is empty in Start Task form', () => {
+
+        tasksCloudDemoPage.openNewTaskForm();
+        startTask.addName(standaloneTaskName2);
```
Comment (eromano) [inline e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts]: standaloneTaskName2 can we find a name that is a characteristic of this task?

please note 1 2 3 or A B C are not good prefix or suffix for names or variable/method/class

File: e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts (hunk 2 above)
Comment (gmandakini) [reply e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts]: not sure :) maybe you can suggest..
screenshot 2019-02-21 at 11 30 34

File: e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts (hunk 3 above)
Comment (gmandakini) [reply e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts]: Updated the names to 'standaloneTaskName' and 'unassignedTaskName' 👍

File: e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts (hunk 2 above)
Comment (eromano) [reply e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts]: use the data-automantion-id go-back>h4

Comment (gmandakini) [reply e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts]: I have added a new data automation id for the task details header.

**Pull request merged**